docs/integrations/cloud/aws-secret-manager-assume-role.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: "AWS Secrets Manager Assume Role"
+description: "Learn how to sync secrets from Infisical to AWS Secrets Manager without sharing any user credentials."
+---
```

Comment:
what if we moved this to the **AWS Secret Manager** doc as a separate tab instead of having a completely separate page for it? 

Since this is also AWS's best practice recommendation, I think we could make it the first tab (and possibly add "We recommend this method for security purposes" somewhere)

<img width="1511" alt="image" src="https://github.com/Infisical/infisical/assets/65645666/0a180977-af88-4241-b83f-809a75d1f0b5">

Reply:
Hmmm, good point.